### ISSUE TITLE
Randomly started crashing after updating.

### ISSUE THREAD

**Issue by Piicrow** [JamSnack/TRandomizer-Mod#3]
Updated all my mods today, from a version probably a month old on each mod. Before the updates I was able to play perfectly fine, but now I'm getting crashes on enable.

Below I've attached errors and logs, as well as my computer's specs and the error on loadup on Terraria. I've also included a screenshot of every mod I've got running. Thanks in advance.

![37b4436839a986a19fd212f01c5d9fb4](https://user-images.githubusercontent.com/61390346/75118265-800bde80-562d-11ea-93c5-c0dbcbe285e7.png)
![ca32e862da3e58913140537946e16838](https://user-images.githubusercontent.com/61390346/75118266-80a47500-562d-11ea-803b-43c492df4d35.png)
![ff06ce7cedc7b67c18554b07822de794](https://user-images.githubusercontent.com/61390346/75118267-813d0b80-562d-11ea-95e2-1036db445cf9.png)

**Comment by JamSnack** [comment]
Hi, the randomizer mod you’re using is not my mod because T-Randomizer only randomizes enemy spawns and chest loot, not NPC drops.

**Comment by Piicrow** [comment]
Yup I realized that after by searching a bit more very sorry about that! I
appreciate the quick response anyway! Have a great day, and I'm excited to
see where you develop your mod further!
